### DISCUSSION TITLE
Any plans to update dokku/docker-grafana-graphite Docker image?

### DISCUSSION THREAD

**Discussion by philbudne** [dokku/dokku#5385] [Q&A]
I almost passed up trying the dokku-graphite plugin after seeing the Docker image was Ubuntu 14.04/Python2 based (at first I hadn't realized that Graphite was still being maintained)!

As my subject line says: Are there any plans to update the default/latest image to something built with software from the current decade?  I'm doing work as a contractor, and leaving my clients with something ancient isn't attractive, and building a local/custom Docker image isn't much better.

If there is anything I can do to advance the effort, please let me know!

Also: is there any way to configure statsd and whisper for 1 minute samples, and longer retention?  In the target environment 6 weeks of 1 minute samples would be much more valuable than 1 week of 6 samples per minute.

Thanks!!!

P.S. I tried to sign up to the slack channel but after entering my email I got a red box with something like "invalid_token_type" before being redirected to the slack login page.

**Comment by josegonzalez** [comment]
There aren't any concrete plans, but if you'd like to contribute a patch to get this working, an issue is available here: https://github.com/dokku/docker-grafana-graphite/issues/7
